### ISSUE TITLE
Wrong syntax generated when a comment is added inside a function

### ISSUE THREAD

**Issue by dilinisg** [ballerinalang/composer#994]
1. Comment the code as below

```
import ballerina.lang.system;
function main(string[] args) {
	string value;
	system:println("Starting....");
	value = args[0];
	//system:println("arguement = " + value);
	
	if (value == "red") {
		sayRed();
		
	}
	else {
		if (value == "orange") {
			sayOrange();
			
		}
		else {
			system:log(5, "This is an invalid value");
			
		}
		
	}
	
}
```

2. Switch between design and source

Issue
Syntax issue

```
import ballerina.lang.system;
function main(string[] args) {
	string value;
	system:println("Starting....");
	value = args[0];
	//system:println("arguement = " + value);;if (value == "red") {
		sayRed();
		
	}
	else {
		if (value == "orange") {
			sayOrange();
			
		}
		else {
			system:log(5, "This is an invalid value");
			
		}
		
	}
	
}

```

**Comment by dilinisg** [comment]
This needs to be fixed in 0.8.1 release

**Comment by hevayo** [comment]
With the new white-space implementation this is not reproducible with 0.88
